src/local-editor-service.ts

```diff
@@ -55,9 +55,25 @@ export class LocalEditorService extends EditorService {
     return feature.sourceRange;
   }
 
-  async getTypeaheadCompletionsAtPosition(
-      localPath: string,
-      position: SourcePosition): Promise<TypeaheadCompletion|undefined> {
+  async getReferencesForFeatureAtPosition(localPath: string,
+                                          position: SourcePosition):
+      Promise<SourceRange[]|undefined> {
+    const document = await this._analyzer.analyze(localPath);
+    const location = await this._getLocationResult(document, position);
+    if (!location) {
+      return;
+    }
+    if (location.kind === 'tagName') {
+      return Array
+          .from(
+              document.getById('element-reference', location.element.tagName!))
```

Comment:
Is this a safe assumption on `tagName`? I assumed so since the `kind` is `tagName`.